bedrock/mozorg/redirects.py

```diff
@@ -588,6 +588,10 @@ def decider(request, **kwargs):
     # bug 1003737
     redirect(r'^impressum/?$', '/about/legal/impressum/'),
 
+    # Bug 682619
+    redirect(r'^support/thunderbird/?$', 'https://support.mozilla.org/products/thunderbird'),
+    redirect(r'^support/firefox/?$', 'https://support.mozilla.org/products/firefox'),
```

Comment:
Same for the firefox redirect, things like `support/firefox/options` should also get redirected here